README.md

```diff
@@ -41,6 +41,7 @@ Other Style Guides
   1. [Naming Conventions](#naming-conventions)
   1. [Accessors](#accessors)
   1. [Events](#events)
+  1. [Standard Library](#standard-library)
```

Comment:
to avoid renumbering all the rest of the sections, let's just add this at the end.